### DISCUSSION TITLE
How change the Inpaint to use color brush again?🤔

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1528] [Q&A] [answered]
I would like to use a color for the brush instead the transparency mode (that squares make feel sick somehow), it's possible to change it back to use a color?

Thx.

**Comment by ZeroCool22** [comment]
![Screenshot_2](https://github.com/user-attachments/assets/77f2db09-c19c-447a-b403-6f76365f0e41)

**Comment by vonespy** [comment]
[trypophobia?](https://www.webmd.com/anxiety-panic/trypophobia-overview)

**Reply by ZeroCool22** [reply]
> [trypophobia?](https://www.webmd.com/anxiety-panic/trypophobia-overview)

![8mx](https://github.com/user-attachments/assets/c7bdbfb1-e9d4-4eb9-97ef-eddbe896142f)

**Reply by vonespy** [reply]
Sorry, I wasn't trying to be funny (for once). Just wondering if that's why checkerboard makes you feel weird. I see a trypophobia button in settings coming soon.

**Comment by Haoming02** [comment, marked as answer]
Hello, you can now disable this checkerboard pattern to use colored brush! ( #1538 )

**Reply by Hujikuio** [reply]
Do you know how to re-add transparency to the colored brush like in reForge / A1111? I usually prefer a white or black brush with some transparency, and if I have to pick between the current options I'd pick the checkerboard so I can see what I'm masking, but it's really hard for me to determine the edges a lot of times.

**Reply by Haoming02** [reply]
<ins>lllyasviel</ins> wrote their own [forge_canvas](https://github.com/lllyasviel/stable-diffusion-webui-forge/tree/main/modules_forge/forge_canvas) for that. So unless you port the entire thing over, you cannot enable the checkerboard pattern easily on Automatic1111.

**Reply by Hujikuio** [reply]
No sorry, I meant the other way around, but I guess the same answer would technically apply? I wanted the ability to set the colored brushes in Forge to have transparency instead of just being solid. Right now for any transparency it's the checkerboard brush or nothing.

**Reply by Haoming02** [reply]
Ah sorry, I misread. And yeah, I just checked, there doesn't seem to be a way to change the transparency of the brush.

**Reply by Haoming02** [reply]
You can add `scribble_alpha` to [ui.py](https://github.com/lllyasviel/stable-diffusion-webui-forge/blob/main/modules/ui.py#L596).

I will go make a PR for this later.

**Reply by Haoming02** [reply]
@Hujikuio The [PR](https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/2461) has been merged~

**Reply by Hujikuio** [reply]
> @Hujikuio The [PR](https://github.com/lllyasviel/stable-diffusion-webui-forge/pull/2461) has been merged~

much love ❤️

**Comment by lakkyninja** [comment]
I have the same problem. i can't switch on the brush. and the chess grid makes me angry! is there any file to replace and restore the brush? please.

![image](https://github.com/user-attachments/assets/c18b95ba-1295-4b30-9083-5bd7ac993444)

**Reply by Haoming02** [reply]
It's already in the settings, since 3 months ago

**Reply by lakkyninja** [reply]
Can you please tell me how to do this?

**Reply by Haoming02** [reply]
**Settings** -> **img2img**

**Reply by lakkyninja** [reply]
wow, next? ![image](https://github.com/user-attachments/assets/16395355-9cab-48fd-b148-27060be9bb5e)

**Reply by Haoming02** [reply]
> next?

Read
